### DISCUSSION TITLE
Remote approve-mode password-click not working

### DISCUSSION THREAD

**Discussion by wmh-se** [rustdesk/rustdesk#9604] [General]
### Bug Description

When choosing the password & click option in remote client 1.3.1 and setting a permanent password, you get an instant connection / session after entering the password. In my understanding the user should first have to approve the connection / session manually.

If i choose the option to approve a session only by click, it works like expected.

### How to Reproduce

In remote client under security set permanent passswort.
Change approval behaviour to password & click.
Connect to remote client from another pc with permanent password.

### Expected Behavior

If the approval behaviour is password & click, both should be nessessary to establish a session.

### Operating system(s) on local side and remote side

Windows 11 -> Windows 11

### RustDesk Version(s) on local side and remote side

1.3.1 -> 1.3.1

### Screenshots

![grafik](https://github.com/user-attachments/assets/d65a0127-f3c2-4d47-bd61-e1aeded27a3b)

**Comment by cvbni** [comment]
I have the same problem，
It work fine for some days，but today the permanent password do not work

**Comment by wmh-se** [comment]
Hi!
I dont get, why this is a discussion now. Did i missunderstand the logic behind click & password approval? (It's not a bug, it's a feature?)

Can anybody clarify?
We are evaluating rustdesk for replacing teamviewer in our company. We want to use a password for security reasons, but most remote users have to approve the connection first. I think it is not such a rare requirement.
Any help?

**Comment by cvbni** [comment]
I set a fixed password, and after a few days, this fixed password will not be able to log in.

**Reply by wmh-se** [reply]
Hi,
it is not the same problem. I never said the permanent password stops working. The click approval (with password) never works for us.
I think you should open a new issue with your problem.
